include/mqt-core/dd/UnaryComputeTable.hpp

```diff
@@ -31,6 +31,8 @@ namespace dd {
 template <class OperandType, class ResultType, std::size_t NBUCKET = 32768>
 class UnaryComputeTable {
 public:
+  static constexpr std::size_t DEFAULT_NUM_BUCKETS = NBUCKET;
+
```

Comment:
That must have been an oversight in my previous PR. The `NBUCKET` template argument no longer makes sense here.
Would you mind removing it and just setting 
```
static constexpr std::size_t DEFAULT_NUM_BUCKETS = 32768U;
```
An then in the constructor below
```
explicit UnaryComputeTable(const size_t numBuckets = DEFAULT_NUM_BUCKETS) {
```